src/js/view/view.js

```diff
@@ -885,7 +896,9 @@ function View(_api, _model) {
 
                 updateFloatingSize();
 
-                _this.floatingUI = new FloatingDragUI(_wrapperElement);
+                if (!_model.get('instreamMode')) {
```

Comment:
could we potentially still drag in outstream mode?

Reply:
`instreamMode` is the mode set to signify that we are playing ads